### DISCUSSION TITLE
Preview apps aren't working for organization members

### DISCUSSION THREAD

**Discussion by mickael-h** [vercel/community#3386] [Feature Question] [answered]
### Summary

I have Vercel connected with my Github organization.
But when a member of my organization pushes a commit on a PR, Vercel asks me to give them access as a unique member for the preview app deployment to work, even though they're already part of that org and therefore should have access.
Does that mean I need to have an extra Vercel seat for each individual member of my Github org, for whom I'm already paying a seat in said Github org, Even though they will NEVER use that Vercel account?

### Steps to Reproduce

Connect your GH org to Vercel.
Have members commits to a PR.

**Comment by amyegan** [comment]
Hey @mickael-h. When using automatic deployment integration with a private repo on a GitHub organization, commit authors must be part of the team. 

> As an additional security measure, commits on private Git repositories (and commits of forks that are targeting those Git repositories) will only be deployed if the commit author also has access to the respective project on Vercel.

> The membership is checked by finding the Personal Account associated with the commit author through comparing [Login Connections](https://vercel.com/docs/accounts/create-an-account#login-methods-and-connections), and then, if a Personal Account is found, checking if it is a member of the Team.

For more info, please visit https://vercel.com/docs/concepts/deployments/git#deploying-private-git-repositories

**Reply by danyel117** [reply]
does it mean that, for example, in a team with 10 developers, you'd need to pay 200usd/month for seeing their development previews?

**Reply by mickael-h** [reply, marked as answer]
@danyel117 I found a workaround: https://github.com/amondnet/vercel-action
It could use better documentation, but it works well.
